src/core-connector-svc/coreConnectorRoutes.ts

```diff
@@ -50,16 +49,17 @@ export class CoreConnectorRoutes implements IRoutes{
         this.routes.push({
            method: ["POST"],
            path: `/quoterequests`,
-           handler: this.quoteRequests.bind(this) 
+           handler: this.quoteRequests.bind(this)
         });
     }
 
-    getRoutes(): ServerRoute<ReqRefDefaults>[] {
+    getRoutes(): ServerRoute[] {
         return this.routes;
     }
 
     private async getParties(request: Request, h: ResponseToolkit){ //todo change to openapi signature
-        this.logger.info("Received GET request /parties/{IdType}/{ID}");
+        // this.logger.info("Received GET request /parties/{IdType}/{ID}");
+        // we don't need this - loggingPlugin will log such info for any incoming requests
```

Comment:
here's an example how it looks like:
![image](https://github.com/mojaloop/mifos-core-connector/assets/16463615/90a9b458-05fd-4b8d-b89f-eeaff0d304b9)

Reply:
yeah looks great actually. Thanks @geka-evk